wolfssl/ssl.h

```diff
@@ -2159,6 +2180,7 @@ enum { /* ssl Constants */
     WOLFSSL_FILETYPE_DEFAULT = 2, /* ASN1 */
     WOLFSSL_FILETYPE_RAW     = 3, /* NTRU raw key blob */
 
+    WOLFSSL_VERIFY_DEFAULT              = -1,
```

Comment:
WOLFSSL_VERIFY_* are bits
WOLFSSL_VERIFY_DEFAULT should have the bits set that reflect what is to be verified by default.

It appears you are actually trying to avoid setting the verifyNone field on the WOLFSSL context. Is that right?

Reply:
I am trying to restore it back to the defaults set before `wolfSSL_CTX_set_verify` is called. I changed how the `WOLFSSL_VERIFY_*` bits are defined and changed `WOLFSSL_VERIFY_DEFAULT` to a higher bit value.

Reply:
Would this be better :
    ssl->options.verifyNone = (mode == WOLFSSL_VERIFY_NONE);
    ssl->options.verifyPeer = ((mode & WOLFSSL_VERIFY_PEER) == WOLFSSL_VERIFY_PEER);
...

Reply:
Changed to your suggested method.